docs/components/RemoteCacheCounter.tsx

```diff
@@ -30,9 +30,9 @@ export default function RemoteCacheCounter() {
   return (
     <Link
       href="/repo/docs/core-concepts/remote-caching"
-      className="group mt-4 w-full h-full rounded-lg p-[1px] bg-gradient-to-r from-red-500 to-blue-500 dark:text-[#9ca3af] text-[#6b7280]"
+      className="mt-4 rounded-lg border border-transparent overflow-hidden bg-origin-border bg-gradient-to-r from-red-500 to-blue-500 dark:text-[#9ca3af] text-[#6b7280]"
```

Comment:
```suggestion
      className="group mt-4 rounded-lg border border-transparent overflow-hidden bg-origin-border bg-gradient-to-r from-red-500 to-blue-500 dark:text-[#9ca3af] text-[#6b7280]"
```

`group` is required for the hover state.